conversant/demo/streamlit_example.py

```diff
@@ -32,7 +33,7 @@ def initialize_chatbot() -> None:
     """Initializes the chatbot from a selected persona and saves the session state."""
     if st.session_state.persona.startswith("from launch_demo") and len(sys.argv) > 1:
         st.session_state.bot = demo_utils.decode_chatbot(
-            sys.argv[1], client=cohere.Client(st.secrets.COHERE_API_KEY)
+            sys.argv[1], client=cohere.Client(os.environ.get("COHERE_API_KEY"))
```

Comment:
ohhh right because streamlit secrets are also injected into the env right? Nice!

Reply:
Yup! Over here: https://github.com/streamlit/streamlit/blob/e1ef414ea4893e8870e8416dceaabd11173ffd00/lib/streamlit/runtime/secrets.py#L197